README.md

```diff
@@ -123,8 +123,23 @@ keystore_password: "my.java.keystore.password"
 github_cert: |
   -----BEGIN CERTIFICATE-----
   -----END CERTIFICATE-----
+
+secrets_py |
+  GCE_PARAMS = ('...@developer.gserviceaccount.com', '/path/to/gce_account.json')
+  GCE_KEYWORD_PARAMS = {'project': 'project_id'}  
+gce_account_certificate_pem |
+  -----BEGIN RSA PRIVATE KEY-----
+  -----END RSA PRIVATE KEY-----
 ```
 
+### Manual steps
+
+Your `jenkins_admin_user` will need to have a `~/.vault_pass.txt` file, the easiest way to create one is to do the following whilst logged in as the `jenkins_admin_user`:
+ 
```

Comment:
Did you actually mean `jenkins` user? This user is defined in the geerlingguy.jenkins ansible role. It's the default jenkins linux user. `jenkins_admin_user` in this readme refers to jenkins users that are also administrators.